package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "big-js": "^3.1.3",
+    "big.js": "^6.2.1",
```

Comment:
- [ ] Could you install one version, the latest one could be the recommendable one

Reply:
Resolved. Thank you